config/vufind/config.ini

```diff
@@ -480,6 +480,7 @@ method         = Database
 ;method         = MultiILS
 ;method         = Facebook
 ;method         = PasswordAccess
+;method         = OpenIDConnect
```

Comment:
We should probably add Open ID Connect to the comment above that lists all possible authentication methods. As part of that comment, we could also reference OpenIDConnect.ini to make it clear that additional configuration is needed. Maybe it would be helpful to reformat the whole comment to something like:

```
; This section allows you to determine how the users will authenticate.
; You can use one or more of the following options:
; - AlmaDatabase (combination of VuFind database and Alma account; see also Alma.ini)
; - CAS (see also [CAS] section)
; - Database: VuFind's internal user database
; - Email (see notes below)
; - Facebook (see also [Facebook] section)
; - ILS: the local ILS
; - LDAP: an LDAP directory (see also [LDAP] section)
; - MultiILS: multiple ILSes (see also MultiBackend.ini)
; - OpenIDConnect: Open ID Connect (see also OpenIDConnect.ini)
; - PasswordAccess: a hard-coded list of access passwords (see also [PasswordAccess] section)
; - Shibboleth (see also [Shibboleth] section)
; - SimulatedSSO: simulated single sign-on for testing/development (see SimulatedSSO.ini)
; - SIP2 (see also [SIP2] section)
; - some combination of the above (via the MultiAuth or ChoiceAuth options).
```

Reply:
I agree, done